schema/openapi_parser_console_test.go

```diff
@@ -358,3 +358,105 @@ func TestKindNotRequiredMetadataField(t *testing.T) {
+		result, err := schema.getRuns(CONSOLE)
+		if err != nil {
+			t.Fatalf("failed getting execute: %s", err)
+		}
+
+		//all the token runs are not present in real life just present in the yaml test file used here
+		expected := RunCatalog{
+			"partnerZoneGenerateCredentials": Run{
+				Path:           "/public/partner-zone/v2/{partner-zone-name}/generate-credentials",
+				Name:           "partnerZoneGenerateCredentials",
+				PathParameter:  []string{"partner-zone-name"},
+				Doc:            "generate a token for a partner zone service account",
+				QueryParameter: map[string]FlagParameterOption{},
+				BodyFields:     map[string]FlagParameterOption{},
+				Method:         "POST",
+				BackendType:    CONSOLE,
+			},
+			"createAdminToken": Run{
+				Path:           "/token/v1/admin_tokens",
+				Name:           "createAdminToken",
+				Doc:            "Create an admin token",
+				QueryParameter: map[string]FlagParameterOption{},
+				PathParameter:  []string{},
+				BodyFields: map[string]FlagParameterOption{
+					"name": {
+						FlagName: "name",
+						Required: true,
+						Type:     "string",
+					},
+				},
+				Method:      "POST",
+				BackendType: CONSOLE,
+			},
+			"createApplicationInstanceToken": Run{
+				Path:           "/token/v1/application_instance_tokens/{application-instance-name}",
+				Name:           "createApplicationInstanceToken",
+				Doc:            "Create an application instance token",
+				QueryParameter: map[string]FlagParameterOption{},
+				PathParameter: []string{
+					"application-instance-name",
+				},
+				BodyFields: map[string]FlagParameterOption{
+					"name": {
+						FlagName: "name",
+						Required: true,
+						Type:     "string",
+					},
+				},
+				Method:      "POST",
+				BackendType: CONSOLE,
+			},
+			"deleteToken": Run{
+				Path:           "/token/v1/{token-id}",
+				Name:           "deleteToken",
+				Doc:            "Delete a token",
+				QueryParameter: map[string]FlagParameterOption{},
+				PathParameter:  []string{"token-id"},
+				BodyFields:     map[string]FlagParameterOption{},
+				Method:         "DELETE",
+				BackendType:    CONSOLE,
+			},
+			"listAdminToken": Run{
+				Path:           "/token/v1/admin_tokens",
+				Name:           "listAdminToken",
+				Doc:            "List admin token",
+				QueryParameter: map[string]FlagParameterOption{},
+				PathParameter:  []string{},
+				BodyFields:     map[string]FlagParameterOption{},
+				Method:         "GET",
+				BackendType:    CONSOLE,
+			},
+			"listApplicationInstanceToken": Run{
+				Path:           "/token/v1/application_instance_tokens/{application-instance-name}",
+				Name:           "listApplicationInstanceToken",
+				Doc:            "List application instance token",
+				QueryParameter: map[string]FlagParameterOption{},
+				PathParameter: []string{
+					"application-instance-name",
+				},
+				BodyFields:  map[string]FlagParameterOption{},
+				Method:      "GET",
+				BackendType: CONSOLE,
+			},
```

Comment:
didn't we wanted only partnerzone run for now ?

Reply:
Yes but this is a test file for the openapi parser doesn't have nothing to do with what is actually shipped at runtime